components/camel-azure/src/main/java/org/apache/camel/component/azure/common/AbstractConfiguration.java

```diff
@@ -17,12 +17,25 @@
 package org.apache.camel.component.azure.common;
 
 import com.microsoft.azure.storage.StorageCredentials;
+import com.microsoft.azure.storage.StorageCredentialsAccountAndKey;
 import org.apache.camel.spi.UriParam;
 
 public abstract class AbstractConfiguration implements Cloneable {
 
+    public static final String MISSING_BLOB_CREDNTIALS_EXCEPTION_MESSAGE =
```

Comment:
Maybe it would be better to remove these messages to the component class or util class and keep configuration class for configuration related stuff?

Reply:
Yes, you're right.